src/cavendish_particle_tracks/_widget.py

```diff
@@ -42,66 +46,90 @@
+            self._on_click_delete_particle
+        )
+        self.btn_radius.clicked.connect(self._on_click_radius)
+        self.btn_length.clicked.connect(self._on_click_length)
+        self.btn_decayangle.clicked.connect(self._on_click_decay_angles)
+        self.btn_stereoshift.clicked.connect(self._on_click_stereoshift)
         self.cal.toggled.connect(self._on_click_apply_magnification)
-        save.clicked.connect(self._on_click_save)
-
-        self.mag.clicked.connect(self._on_click_magnification)
+        self.btn_save.clicked.connect(self._on_click_save)
+        self.btn_magnification.clicked.connect(self._on_click_magnification)
         # TODO: find which of thsese works
         # https://napari.org/stable/gallery/custom_mouse_functions.html
         # self.viewer.mouse_press.callbacks.connect(self._on_mouse_press)
         # self.viewer.events.mouse_press(self._on_mouse_click)
 
         # layout
-        self.setLayout(QVBoxLayout())
-        self.layout().addWidget(self.load)
-        self.layout().addWidget(self.cb)
-        self.layout().addWidget(delete_particle)
-        self.layout().addWidget(self.rad)
-        self.layout().addWidget(self.lgth)
-        self.layout().addWidget(self.ang)
+        self.viewer.window._qt_viewer.layerButtons.hide()  # This will break in napari 0.6.0
+        self.buttonbox = QGridLayout()
+        self.buttonbox.addWidget(self.btn_load, 0, 0)
+        self.buttonbox.addWidget(self.cmb_add_particle, 1, 0)
+        self.buttonbox.addWidget(self.btn_delete_particle, 1, 1)
+        self.buttonbox.addWidget(self.btn_radius, 2, 0)
+        self.buttonbox.addWidget(self.btn_length, 2, 1)
+        self.buttonbox.addWidget(self.btn_decayangle, 3, 0)
+        self.buttonbox.addWidget(self.btn_stereoshift, 3, 1)
+        self.buttonbox.addWidget(self.btn_magnification, 4, 0)
+        self.buttonbox.addWidget(self.cal, 4, 1)
+        self.buttonbox.addWidget(self.btn_save, 5, 0)
+
+        layout_outer = QHBoxLayout()
+        self.setLayout(layout_outer)
+        self.intro_text = QLabel(
+            r"""
+   _____                          _ _     _       _____           _   _      _        _______             _
+  / ____|                        | (_)   | |     |  __ \         | | (_)    | |      |__   __|           | |
+ | |     __ ___   _____ _ __   __| |_ ___| |__   | |__) |_ _ _ __| |_ _  ___| | ___     | |_ __ __ _  ___| | _____
+ | |    / _` \ \ / / _ \ '_ \ / _` | / __| '_ \  |  ___/ _` | '__| __| |/ __| |/ _ \    | | '__/ _` |/ __| |/ / __|
+ | |___| (_| |\ V /  __/ | | | (_| | \__ \ | | | | |  | (_| | |  | |_| | (__| |  __/    | | | | (_| | (__|   <\__ \
+  \_____\__,_| \_/ \___|_| |_|\__,_|_|___/_| |_| |_|   \__,_|_|   \__|_|\___|_|\___|    |_|_|  \__,_|\___|_|\_\___/
+
+Copyright (c) 2023-24 Sam Cunliffe and Paula Álvarez Cartelle 2024 Joseph Garvey under the MIT License.
+"""
+        )
+        print(self.intro_text.text())
+        self.intro_text.setFont(QFont("Consolas", 5))
```

Comment:
On MacOS this looks a bit weird. Maybe related with this warning
```
WARNING: Populating font family aliases took 158 ms. Replace uses of missing font family "Consolas" with one that exists to avoid this cost.
```
![Screenshot 2024-09-10 at 15 02 22](https://github.com/user-attachments/assets/e6d108ae-6778-4b9c-8390-7f0ba69fc7fc)

Reply:
No rush to fix this.